appium/api-mocks/core/msg-block-parser.ts

```diff
@@ -112,22 +64,24 @@ export class MsgBlockParser {
     return normalized;
   }
 
-  public static extractFcReplyToken = (decryptedContent: string): undefined | any => {
-    const fcTokenElement = $(`<div>${decryptedContent}</div>`).find('.cryptup_reply');
-    if (fcTokenElement.length) {
-      const fcData = fcTokenElement.attr('cryptup-data');
-      if (fcData) {
-        return Str.htmlAttrDecode(fcData);
-      }
-    }
-  }
+  // public static extractFcReplyToken = (decryptedContent: string): undefined | any => {
+  //   // todo - used exclusively on the web - move to a web package
+  //   const fcTokenElement = $(`<div>${decryptedContent}</div>`).find('.cryptup_reply');
+  //   if (fcTokenElement.length) {
+  //     const fcData = fcTokenElement.attr('cryptup-data');
+  //     if (fcData) {
+  //       return Str.htmlAttrDecode(fcData);
+  //     }
+  //   }
+  // }
 
   public static stripFcTeplyToken = (decryptedContent: string) => {
     return decryptedContent.replace(/<div[^>]+class="cryptup_reply"[^>]+><\/div>/, '');
   }
 
   private static isFcAttachmentLinkData = (o: any): o is FcAttachmentLinkData => {
-    return o && typeof o === 'object' && typeof (o as FcAttachmentLinkData).name !== 'undefined' // tslint:disable-line:no-unsafe-any
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+    return o && typeof o === 'object' && typeof (o as FcAttachmentLinkData).name !== 'undefined'
```

Comment:
How about we do this way?
```ts
  private static isFcAttachmentLinkData = (o: FcAttachmentLinkData): o is FcAttachmentLinkData => {
    return o && typeof o === 'object' && typeof (o as FcAttachmentLinkData).name !== 'undefined'
      && typeof (o as FcAttachmentLinkData).size !== 'undefined' && typeof (o as FcAttachmentLinkData).type !== 'undefined';
  }
```

Reply:
This code was used only in `extractFcAttachments`, which isn't used anywhere, so I just removed it.